sdk/storage/storage-file-datalake/test/perfstress/track-2/append.spec.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PerfStressOptionDictionary } from "@azure/test-utils-perfstress";
+import { StorageDFSTest } from "./storageTest.spec";
+import { DataLakeFileClient } from "../../../src";
+import { generateUuid } from "@azure/core-http";
+interface StorageDFSAppendTestOptions {
+  size: number;
+}
+
+export class StorageDFSAppendTest extends StorageDFSTest<StorageDFSAppendTestOptions> {
+  initialBuffer = Buffer.alloc(0);
+  buffer = Buffer.alloc(this.parsedOptions.size.value!);
```

Comment:
I wonder if we should generate random data for the buffer content. Though we don't seem to have any optimization for all-zeros contents in the client size.

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/13501 logged a separate issue to address all the packages.
Not doing in this PR.